admin/templates/configuration/etc/init.d/qserv-czar

```diff
@@ -46,6 +49,17 @@ export QSW_RESULTDIR=${QSERV_RUN_DIR}/qserv-run/tmp
 # Disabling buffering in python in order to enable "real-time" logging.
 export PYTHONUNBUFFERED=1
 
+export XRD_LOGLEVEL=Debug
+export XRDSSIDEBUG=1
+
+# Increase timeouts, without that, long-running queries fail.
+# The proper fix is coming, see DM-3433
+export XRD_REQUESTTIMEOUT=64000
+export XRD_STREAMTIMEOUT=64000
+export XRD_DATASERVERTTL=64000
+export XRD_TIMEOUTRESOLUTION=64000
+
+
```

Comment:
Fabrice. I have nothing against it, but... we are doing plenty of similar things already in init.d/qserv-czar and init.d/zookeeper, and I'd rather not do this sort of surgery right now (partly because I am not entirely clear how to transport things to QSERV_RUN_DIR/etc/). If you can quickly send me some directions I will try to implement it tomorrow (Thu), otherwise we will deal with that in the future. So, should I put it in ./templates/configuration/etc/sysconfig/ and call a "cp" in some other script?